src/dependency-graph.ts

```diff
@@ -28,7 +28,7 @@ export async function compare({
   const changes = await octo.paginate(
     {
       method: 'GET',
-      url: '/repos/{owner}/{repo}/dependency-graph/compare/{basehead}',
+      url: '/repos/{owner}/{repo}/dependency-graph/compare/{basehead}?includes_dependency_snapshots=true',
```

Comment:
This param is unnecessary now, it's controlled by the feature flag.

```suggestion
      url: '/repos/{owner}/{repo}/dependency-graph/compare/{basehead}',
```